low-level/files/opt/seagate/sspl/bin/pacemaker/README

```diff
@@ -8,15 +8,15 @@
     - Make sure consul service is running on nodes.
 
     - If system is not provisioned, execute following steps to initialize SSPL:
-        /opt/seagate/cortx/sspl/bin/sspl_post_install -p LDR_R1
-        /opt/seagate/cortx/sspl/bin/sspl_setup_init -r cortx
-        /opt/seagate/cortx/sspl/bin/sspl_config
+        /opt/seagate/cortx/sspl/low-level/files/opt/seagate/sspl/bin/sspl_post_install -p LDR_R1
```

Comment:
same comments, want deployment paths to remain same

Reply:
Done